# HG changeset patch
# User Unity
# Date 1405788613 14400
# Node ID 28422d4c9ca08c5dc80ea1d2bead762c3dc4bfad
# Parent  ba41357756628364447abe78b0e675aca7496030
Mod playerbot to not show inventory on mouse-over in patched-base.

diff --git a/patched-base/src/game/playerbot/PlayerbotMgr.cpp b/patched-base/src/game/playerbot/PlayerbotMgr.cpp
--- a/patched-base/src/game/playerbot/PlayerbotMgr.cpp
+++ b/patched-base/src/game/playerbot/PlayerbotMgr.cpp
@@ -288,12 +288,14 @@
         // such as inventory that can be equipped
     case CMSG_INSPECT:
         {
+            /* >>> Moved to textemote /curious - was too annoying
             WorldPacket p(packet);
             p.rpos(0); // reset reader
             ObjectGuid guid;
             p >> guid;
             Player* const bot = GetPlayerBot(guid);
             if (bot) bot->GetPlayerbotAI()->SendNotEquipList(*bot);
+            */
             return;
         }
 
@@ -379,6 +381,18 @@
                     return;
                 }
 
+
+            // emote to show inventory that can be equipped
+            case TEXTEMOTE_CURIOUS:
+                {
+                    WorldPacket p(packet);
+                    p.rpos(0); // reset reader
+                    ObjectGuid guid;
+                    p >> guid;
+                    Player* const bot = GetPlayerBot(guid);
+                    if (bot) bot->GetPlayerbotAI()->SendNotEquipList(*bot);
+                }
+
                 // emote to attack selected target
             case TEXTEMOTE_POINT:
                 {

